backoff.py

```diff
@@ -218,10 +218,22 @@ def constant(interval=1):
         yield interval
 
 
+def random_jitter(value):
+    return value+random.random()
```

Comment:
You're right. I thought there always had to be a space on each side of the operator. I guess not!